### PR TITLE
AIOPS-1868 Change generated labels

### PR DESCRIPTION
**Pull request by ajtuss**

### CODE REVIEW

File: batch.go [hunk 1]
```diff
@@ -243,14 +233,30 @@ func newLabelPool(faker *fake.Faker, cardinalities map[string]int) LabelPool {
 		"format": []string{"apache_common", "apache_combined", "apache_error", "rfc3164", "rfc5424", "json", "logfmt"}, // needs to match the available flog formats
 		"os":     []string{"darwin", "linux", "windows"},
 	}
+
+	num := 10
+	var nss []string
```
Comment (arempter) [inline batch.go]: no need to declare this. just add : in line 242
nss := generateValues(faker.BS, num)

File: batch.go [hunk 2]
```diff
@@ -243,14 +233,30 @@ func newLabelPool(faker *fake.Faker, cardinalities map[string]int) LabelPool {
...
 	if n, ok := cardinalities["namespace"]; ok {
```
Comment (arempter) [inline batch.go:239]: this part is confusing to me, not sure why you use these default values here again. they are defined in
https://github.com/co-brick/xk6-loki/blob/main/loki.go#L160
so if len(labels) == 0 they will be initialized anyway

File: batch.go [hunk 3]
```diff
@@ -243,14 +233,30 @@ func newLabelPool(faker *fake.Faker, cardinalities map[string]int) LabelPool {
...
-		lb["namespace"] = generateValues(faker.BS, n)
+		num = n
 	}
+	nss = generateValues(faker.BS, num)
+	lb["namespace"] = nss
+	num = 5
```
Comment (arempter) [inline batch.go]: like above I think no need to have default here

File: batch.go [hunk 4]
```diff
@@ -243,14 +233,30 @@ func newLabelPool(faker *fake.Faker, cardinalities map[string]int) LabelPool {
...
 	if n, ok := cardinalities["app"]; ok {
-		lb["app"] = generateValues(faker.AppName, n)
+		num = n
 	}
+	svs := generateValues(fakeAppName(faker), num)
+	lb["app"] = svs
+	var jobs []string
+	for _, ns := range nss {
```
Comment (arempter) [inline batch.go]: just guessing, but this loop has potential for duplicate labels

Comment (ajtuss) [reply batch.go]: These loops create `job` labels from generated namespaces and services and join them to the namespace/service pattern. But assigning `app` in 249 could duplicate labels, so I deleted it.

**Pull request merged**